ui/build-ui-plugins.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
yeah i did sneak in some extra stuff here because i thought it would be nice to address lack of ui build logs getting in google cloud. will do better next time